dotcom-rendering/src/web/components/Caption.tsx

```diff
@@ -147,6 +147,15 @@ const hideIconBelowLeftCol = css`
 	}
 `;
 
+const videoCaptionOverlayStyles = css`
```

Comment:
Is this change related to repositioning the main media?

Reply:
Yes, it is. As the video has moved up, we now don't have the caption below the video. Instead, it appears as an overlay on the top of the video. Here's the design in a screengrab
<img width="901" alt="Screenshot 2022-08-10 at 09 33 58" src="https://user-images.githubusercontent.com/20416599/183854919-eb00f451-3f89-463d-9d96-a1264e043787.png">